internal/driver/glfw/loop.go

```diff
@@ -188,26 +190,25 @@ func (d *gLDriver) destroyWindow(w *window, index int) {
 func (d *gLDriver) repaintWindow(w *window) bool {
```

Comment:
It seems like the contract for this function changed so that the caller is now responsible for calling it with GL context? I think there may be one call path where this isn't the case (window.Resize -> platformResize)

Reply:
Thanks for that - I misread main and context run helpers!